distribution/src/resources/config-tool/deployment-full.toml

```diff
@@ -646,6 +646,69 @@ parameter.queue_auto_declare = ""
 parameter.exchange_auto_declare = ""
 parameter.connection_pool_size = 10
 
+[transport.blocking.rabbitmq]     # RabbitMQ blocking transport
+sender_enable = false
+
+[[transport.blocking.rabbitmq.sender]]
+
+name = "AMQPConnectionFactory"
+parameter.hostname = "localhost"
+parameter.port = 5672
+parameter.username = "guest"
+parameter.password = "guest"
+parameter.exchange_name = "amq.direct"
+parameter.routing_key = "MyQueue"
+parameter.reply_to_name = ""
+parameter.queue_delivery_mode = 1 # 1/2
+parameter.exchange_type = ""
+parameter.queue_name = "MyQueue"
+parameter.queue_durable = false
+parameter.queue_exclusive = false
+parameter.queue_auto_delete = false
+parameter.exchange_durable = ""
+parameter.queue_auto_declare = ""
+parameter.exchange_auto_declare = ""
+parameter.connection_pool_size = 10
+
+[transport.blocking.rabbitmq]
+listener_enable = true
```

Comment:
Shall we move this section under the above same [transport.blocking.rabbitmq] section?

Reply:
Fixed in fff57307cbc5a2cddf98e4ded1182b7e18e58098